operations_on_datastructures/trie_multiple_search.cpp

```diff
@@ -0,0 +1,471 @@
+/**
+ * @file
+ * @brief [Trie datastructure](https://iq.opengenus.org/autocomplete-using-trie-data-structure/)
+ * with search variants
+ * @details
+ * This provides multiple variants of search functions
+ * on a trie structure utilizing STL. The trie is valid
+ * for only English alphabets.
+ * @author [Ghanashyam](https://github.com/g-s-k-zoro)
+ */
+
+#include <cassert>   /// for assert
+#include <cctype>    /// for tolower
+#include <cstring>   /// for string operations
+#include <iostream>  /// for IO Operations
+#include <queue>     /// for std::priority_queue
+
+/**
+ * @namespace operations_on_datastructures
+ * @brief Operations on data structures
+ */
+namespace operations_on_datastructures {
+/**
+ * @namespace trie_operations
+ * @brief Functions for [Trie datastructure](https://iq.opengenus.org/autocomplete-using-trie-data-structure/)
+ * implementation
+ */
+namespace trie_operations {
+

```

Comment:
```suggestion
```

Reply:
In CONTRIBUTING.md there is a newline after namespace and before class description. Should I change it anyway?

Reply:
Yes, if you want you can change it. 🙂

Reply:
Done. Removed the newline. Any other changes ?